[rocclr] Build failure: hsa-runtime64 not found

As so:

```
==> Starting build()...
-- The C compiler identification is GNU 10.2.0
-- The CXX compiler identification is GNU 10.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Code Object Manager found at /opt/rocm/lib/cmake/amd_comgr.
CMake Error at CMakeLists.txt:55 (find_package):
  Could not find a package configuration file provided by "hsa-runtime64"
  with any of the following names:

    hsa-runtime64Config.cmake
    hsa-runtime64-config.cmake

  Add the installation prefix of "hsa-runtime64" to CMAKE_PREFIX_PATH or set
  "hsa-runtime64_DIR" to a directory containing one of the above files.  If
  "hsa-runtime64" provides a separate development package or SDK, be sure it
  has been installed.


-- Configuring incomplete, errors occurred!
See also "/tmp/yaourt-tmp-koz/aur-rocclr/src/build/CMakeFiles/CMakeOutput.log".
==> ERROR: A failure occurred in build().
    Aborting...
==> ERROR: Makepkg was unable to build rocclr.
```

--- a/rocclr/.SRCINFO
+++ b/rocclr/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = rocclr
 	pkgdesc = Radeon Open Compute Common Language Runtime
 	pkgver = 3.7.0
-	pkgrel = 1
+	pkgrel = 2
 	url = https://github.com/ROCm-Developer-Tools/ROCclr
 	arch = x86_64
 	license = MIT

--- a/rocclr/PKGBUILD
+++ b/rocclr/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer Torsten Keßler <t dot kessler at posteo dot de>
 pkgname=rocclr
 pkgver=3.7.0
-pkgrel=1
+pkgrel=2
 pkgdesc='Radeon Open Compute Common Language Runtime'
 arch=('x86_64')
 url='https://github.com/ROCm-Developer-Tools/ROCclr'
@@ -26,5 +26,6 @@ build() {
 
 package() {
     DESTDIR="$pkgdir" make -C build install
+    sed -i "s@$srcdir/build@/opt/rocm/rocclr@" "$pkgdir/opt/rocm/rocclr/lib/cmake/rocclr/ROCclrConfig.cmake"
     install -Dm644 "$_dirname/LICENSE.txt" "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
 }

